app/models/spree/user.rb

```diff
@@ -49,14 +45,14 @@ def admin?
       has_spree_role?('admin')
     end
 
-    def send_reset_password_instructions
```

Comment:
where did this go then?

Reply:
We need to get our code review process faster, I mean, this was written 1 month ago, I am not the author anymore.
Let's see what this Luis from one month ago wanted to do.... I'll investigate now.

Reply:
ok, got it. this is now done in vanilla devise:
https://rubydoc.info/github/plataformatec/devise/master/Devise/Models/Recoverable/ClassMethods#send_reset_password_instructions-instance_method

It comes into our User.rb through the recoverable decorator :+1: and we dont need to change it on our side anymore.